paxos/src/endpoint/assetAttestations.ts

```diff
@@ -0,0 +1,52 @@
+import { Requester } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+import { BASE_URL } from '../config'
+
+export const Name = 'assetAttestation'
+
+const customError = (data: any) => data.Response === 'Error'
+
+type RequestData = {
+  asset: string
+}
+
+export const inputParams = {
+  asset: true,
+}
+
+type Attestation = {
+  asset: string
+  auditorName: string
+  lastAttestedAt: string
+  amount: number
+  verified: boolean
+}
+
+const getAttestationURI = (asset: string) => `/asset-attestations/${asset.toUpperCase()}`
+
+const toAttestation = async (config: Config, asset: string): Promise<Attestation> => {
+  const url = getAttestationURI(asset)
+  const params = {}
```

Comment:
Could remove this here and the next line unless you are intending to clear out `params`